### PR TITLE
[Bugfix]Frontpage icon alignment with circles

### PR DESCRIPTION
**Pull request by LJKaski**
Icons are also automatically resized to fit the circles if resized in code. Could do a component implementation of this later.

### CODE REVIEW

File: src/components/ContentBoxes.tsx [hunk 1]
```diff
@@ -14,14 +17,16 @@ const Block = ({ block }: { block: Block }): JSX.Element => (
...
+          width: iconBubbleSize + 'px',
+          height: iconBubbleSize + 'px',
           marginBottom: suomifiTheme.spacing.l,
           border: `1px solid ${suomifiTheme.colors.depthLight13}`,
           borderRadius: '50%',
-          fontSize: '50px'
+          fontSize: iconsize + 'px'
         }}>
-        <Icon icon={block.icon} />
+        <div style={{ marginBottom: '-0.1em' }}>
```
Comment (aappoalander) [inline src/components/ContentBoxes.tsx]: Should we fix this in the ui-components repo? For some reason there is a transform: translateY(0.1em); defined in Icon.baseStyles.tsx, not sure why. Maybe @J-Kallunki knows?

Comment (aappoalander) [reply src/components/ContentBoxes.tsx]: As discussed elsewhere, I'd propose to target the problematic style directly to icon with e.g. transform:none and remove it once this issue has been resolved in ui-components repo.

**Comment by ketsappi** [comment]
If it possible to fix just the "people" icon separately? As it seems now, the other two ones are no not so well aligned to center after the change.

After
after

Before
before

**Comment by LJKaski** [comment]
This is deprecated/obsolete since the issue was fixed in the ui-components repo as proposed in the comments.

**Pull request closed without merging**